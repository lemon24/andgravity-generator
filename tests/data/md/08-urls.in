[1](http://example/one)
[2](attachment:two)
![3](http://example/three)
![4](attachment:four)
