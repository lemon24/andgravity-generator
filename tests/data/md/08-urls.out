<p><a href="http://example/one">1</a>
<a href="ATTACHMENT:TWO">2</a>
<img src="http://example/three" alt="3" />
<img src="ATTACHMENT:FOUR" alt="4" /></p>
